src/qt/bitcoin.cpp

```diff
@@ -487,13 +487,12 @@ void BitcoinApplication::initializeResult(bool success)
         }
 #endif
 
-        // If -min option passed, start window minimized.
-        if(gArgs.GetBoolArg("-min", false))
-        {
-            window->showMinimized();
-        }
-        else
-        {
+        // If -min option passed, start window minimized or minimized to tray
+        if (gArgs.GetBoolArg("-min", false)) {
```

Comment:
I think it would be more clear doing like:
```cpp
if (!gArgs.GetBoolArg("-min", false)) {
    window->show();
} else if (window->hasTrayIcon() && clientModel->getOptionsModel()->getMinimizeToTray()) {
    // do nothing as the window is managed by the tray icon
} else {
    // show window minimized as requested
}
```